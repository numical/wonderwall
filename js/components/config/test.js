'use strict';
/* eslint-env jasmine */

import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import ConfigPage from './index.js';

describe('Config Page ', () => {
  it('should match previous snapshot', () => {
    const page = shallow(<ConfigPage />);
    expect(shallowToJson(page)).toMatchSnapshot();
  });

  it('should say Config Page', () => {
    const page = shallow(<ConfigPage />);
    expect(page.find('div').text()).toEqual('Config Page');
  });
});

