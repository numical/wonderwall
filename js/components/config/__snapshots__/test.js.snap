exports[`Config Page  should match previous snapshot 1`] = `
<div>
  Config Page
</div>
`;
